WooCommerce/WooCommerceTests/Authentication/Store Creation/DefaultStoreCreationStoreSwitchSchedulerTests.swift

```diff
@@ -0,0 +1,102 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+
+@MainActor
+final class DefaultStoreCreationStoreSwitchSchedulerTests: XCTestCase {
+    func test_isPendingStoreSwitch_is_true_when_pending_store_switch() throws {
```

Comment:
Nit: We should write with proper grammar to make it easier to read.

```suggestion
    func test_isPendingStoreSwitch_is_true_when_there_exists_pending_store_switch() throws {
```

Same comment for the following test cases.

Reply:
Done in 41ea3c4617da7b15b3439afe01271a3d052a567a